data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/vnode-vault-creation-v5.1.xml

```diff
@@ -6,5 +6,11 @@
         <addColumn tableName="vnode_member_info">
             <column name="is_deleted" type="BOOLEAN" defaultValueBoolean="false"/>
         </addColumn>
+
+        <createTable tableName="vnode_persistence_request_id">
+            <column name="request_id" type="NVARCHAR(255)">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+        </createTable>
```

Comment:
What is the cleanup strategy for this table?
Would it not be useful to have a timestamp column that can be used for partitioning and/or cleanup?
Given it is insert only, I suspect it would be very cheap to have a DB auto-populated timestamp column, but we could check with @ifrankrui

Reply:
But do we have a design yet for the clean-up strategy?
I'm a bit uncomfortable with merging something in which we know will cause problems, without a plan for how we will mitigate it.
And if we know what the design will be, we may as well get the DB structure correct from the get-go.

Reply:
Don't you already include the housekeeping strategy [here](https://github.com/corda/platform-eng-design/pull/568)? 

Yes, you could have a timestamp column for the deduplication table and populate the timestamp as a part of the insert request. However, I would not recommend using DB auto-populated timestamp column. They are implemented differently between databases and there isn't one simple way to define them in Liquibase change log.

Reply:
> there isn't one simple way to define them in Liquibase change log.

@ifrankrui I'm interested in this comment. Why is that?
We do this in a number of places already (e.g.: https://github.com/corda/corda-api/blob/9ea985216d72b74f946d9f967699244bce97c34b/data/db-schema/src/main/resources/net/corda/db/schema/config/migration/config-creation-v1.0.xml#L6)

Reply:
@driessamyn As pointed out in the above changelog, you have to define an extra property for one type of database when you support multiple database types. If you populate the timestamp column as a part of a persistence request, then you don't need these Liquibase properties.